src/main.rs

```diff
@@ -170,6 +171,16 @@ async fn main() -> Result<()> {
         let state = paths::load_state_file().await?;
         if should_update(&args, state.as_ref()).await? {
             update(&args, state.as_ref(), &install_path, download_attempts).await?;
+
+            // Launch post exec hook script if configured
+            match &cf.spotify.post_update_exec_hook {
```

Comment:
This can be written as `if let Some(path) = &cf.spotify.post_update_exec_hook {` :)